templates/server.erb

```diff
@@ -46,7 +46,7 @@ dh <%= @etc_directory -%>/openvpn/<%= @ca_name %>/keys/dh.pem
 crl-verify <%= @etc_directory -%>/openvpn/<%= @ca_name %>/crl.pem
 <%   end -%>
 <% end -%>
-<% if @proto == 'tcp' -%>
+<% if @proto != 'udp' -%>
```

Comment:
is this still correct? Or should it be something like this (untested):
```suggestion
<% unless ['udp', 'udp4', 'udp6'].include?(@proto) -%>
```

Reply:
> is this still correct?

No, I found this issue while I wrote the test. 😃